.ci/atime/tests.R

```diff
@@ -108,7 +108,7 @@ test.list <- atime::atime_test_list(
 
   # Issue reported in: https://github.com/Rdatatable/data.table/issues/5426
   # To be fixed in: https://github.com/Rdatatable/data.table/pull/5427
-  "setDT improved in #5427" = atime::atime_test(
+  "setDT improved fixed in #5427" = atime::atime_test(
```

Comment:
Please revert the change here (the altered wording sounds wrong and this is a separate test)

Reply:
this is not resolved, please change "improved fixed" back to "improved"